content/systems/filecoin_markets/retrieval_market/_index.md

```diff
@@ -9,89 +9,75 @@ weight: 2
-- The provider unseals the sector as necessary
-- The provider monitors data transfer as it sends blocks over the protocol, until it requires payment
-- When the provider requires payment, it pauses the data transfer and sends a request for payment as an intermediate voucher
-- The client receives the request for payment
-- The client creates and stores payment voucher off-chain
-- The client responds to provider with a reference to the payment voucher, sent as an intermediate voucher
-- The provider redeems the payment voucher off-chain
-- The provider resumes both the request and sending data
-- The process continues until the end of the query
+- The provider validates the proposal and rejects it if it is invalid.
+- If the proposal is valid, the provider responds with an accept message and begins monitoring the data transfer process.
+- The client creates a payment channel as necessary and a "lane" and ensures there are enough funds in the channel.
+- The provider unseals the sector as necessary.
+- The provider monitors data transfer as it sends blocks over the protocol, until it requires payment.
+- When the provider requires payment, it pauses the data transfer and sends a request for payment as an intermediate voucher.
+- The client receives the request for payment.
+- The client creates and stores a payment voucher off-chain.
+- The client responds to the provider with a reference to the payment voucher, sent as an intermediate voucher (i.e., acknowledging receipt of a part of the data and channel or lane value).
+- The provider redeems the payment voucher off-chain.
+- The provider resumes sending data and requesting intermediate payments.
+- The process continues until the end of the data transfer.
+
+Some extra notes worth making with regard to the above process are as follows:
+
+- The payment channel is created by the client.
+- The payment channel is not created until the provider accepts the deal.
+- The vouchers are also created by the client and (a reference/identifier to these vouchers is) sent to the provider.
+- The payment indicated in the voucher is not taken out of the payment channel funds upon creation and exchange of vouchers between the client and the provider.
+- In order for money to be transferred to the provider's payment channel side, the provider has to *redeem* the voucher
+- In order for money to be taken out of the payment channel, the provider has to submit the voucher on-chain and `Collect` the funds.
+- Both redeeming and collecting vouchers/funds can be done at any time during the data transfer, but collecting funds involves the blockchain, which means that it incurs gas cost.
+- Once the data transfer is complete, there is a 12hr period within which the provider has to submit the redeemed vouchers on-chain in order to collect the funds. Otherwise, the client is free to close the channel, in which case, the provider loses the funds.
```

Comment:
Once the data transfer is complete, the client or provider may Settle the channel. There is then a 12hr period within which the provider has to submit the redeemed vouchers on-chain in order to collect the funds. Once the 12hr period is complete, the client may collect any unclaimed funds from the channel, and the provider loses the funds for vouchers they did not submit.